IPython/extensions/tests/test_autoreload.py

```diff
@@ -132,7 +132,6 @@ class TestAutoreload(Fixture):
 
     @skipif(sys.version_info < (3, 6))
     def test_reload_enums(self):
-        import enum
```

Comment:
@Carreau you wrote this test; is it possible that it relies on enum being imported before the rest of the test runs, for some reason? I know that testing autoreload can be a bit fiddly.